concourse/pipelines/pull-request.yml

```diff
@@ -2,7 +2,7 @@
 definitions:
   main_python_version: &main_python_version "3.9.6"
   python_versions: &python_versions ["3.6.14", "3.7.11", "3.8.11", "3.9.6"]
-  databases: &databases ["sqlite", "postgresql", "mysql", "mariadb"]
```

Comment:
:warning: look like you did fix on develop but proposed to commit into maintenance/4.0.

Reply:
I don't think there is a risk for an unexpected impact in maintenance/4.0 with the commits we did here if we merge

Reply:
so should I just remove this line?

Reply:
As said orally I would rather not get those commits from develop here:
- they are not required for the maintenance branch.
- as they will be squashed it could lead to future confilcts when maintenance/4.0 is merged back.
- it will be good practice to learn rebasing :-)